common/src/main/java/com/datatorrent/common/util/PubSubWebSocketClient.java

```diff
@@ -376,7 +385,8 @@ public void unsubscribe(String topic) throws IOException
    * @return subscribe num subscriber message.
    * @throws IOException
    */
-  public static <T> String constructSubscribeNumSubscribersMessage(String topic, PubSubMessageCodec<T> codec) throws IOException
+  public static <T> String constructSubscribeNumSubscribersMessage(String topic, PubSubMessageCodec<T> codec) throws
+      IOException
```

Comment:
David I made those changes. However I have some comments here:
About your consistency point: places where throws may have a break before because the method name and arguments maybe long enough.
And as I mentioned before if this task it to ensure readability of classes that I am not the author of, then I shouldn't be making these changes. Individual authors/owners should take them up.